frontend/app-development/features/appPublish/pages/DeployPage.tsx

```diff
@@ -35,9 +36,9 @@ export function DeployPage() {
 
   if (orgsIsError || permissionsIsError)
     return (
-      <Alert severity='danger' className={classes.alert}>
-        {t('app_deployment.error')}
-      </Alert>
+      <div className={classes.alert}>
+        <StudioError>{t('app_deployment.error')}</StudioError>;
+      </div>
```

Comment:
Ref. previous comments about exposing the API for the `Alert` component. Doing so could revert the changes due to the `className` here and avoid one `div` node in the DOM. 😊  
```suggestion
        <StudioError className={classes.alert}>{t('app_deployment.error')}</StudioError>;
```